mesheryctl/internal/cli/root/system/reset.go

```diff
@@ -80,6 +80,13 @@ func resetMesheryConfig() error {
 		if currChannel == "edge" {
 			fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
 		} else if currChannel == "stable" {
+			if currVersion == "" {
+				log.Info("Version unspecified, fetching latest stable release...\n")
```

Comment:
We may not need to log this to the user. It would be confusing.

Since we output the version at the end, I don't think we need this. If this looks good to everyone, then don't mind me.

Reply:
I was also a little confused about this while adding it. Seemed a little verbose. Let's see what others suggest 👀.

Reply:
Yes, that is what I thought. Maybe this could be a debug(verbose) log?

Reply:
removed it ✔